src/schemas/21-526EZ/schema.js

```diff
@@ -36,9 +40,46 @@ const disabilitiesBaseDef = {
   }
 };
 
-
-// Extracted to enable easy adding of properties for forwardingAddress
-const addressDef = definitions.pciuAddress;
+const addressBaseDef = {
+  type: 'object',
+  required: ['country', 'city', 'addressLine1'],
+  properties: {
+    country: {
+      type: 'string',
+      'enum': pciuCountries,
+      default: 'USA'
+    },
+    addressLine1: {
+      type: 'string',
+      maxLength: 35,
+      pattern: "([a-zA-Z0-9-'.,,&#]([a-zA-Z0-9-'.,,&# ])?)+$"
```

Comment:
Should we perhaps prefix these patterns with `^` even though EVSS doesn't? Also...because this is living in vets-website, I think we should clean up those extra `,`s as well. (As much as I'd like to do more, I think we should probably leave the refactoring after that.)

Reply:
Yeah, that's a good call.